apps/lend/src/components/ChartOhlcWrapper/PoolActivity.tsx

```diff
@@ -31,12 +25,16 @@ const PoolActivity = ({ chainId, poolAddress, coins }: Props) => {
 
   const [eventOption, setEventOption] = useState<'TRADE' | 'LP'>('TRADE')
 
+  const minHeight = useMemo(() => {
```

Comment:
useMemo is not necessary in such a simple operation, a normal variable declaration is sufficient